runtime/runtime_controller.h

```diff
@@ -22,13 +22,89 @@
 #include "rapidjson/stringbuffer.h"
 
 namespace flutter {
+
 class Scene;
 class RuntimeDelegate;
 class View;
 class Window;
 
+//------------------------------------------------------------------------------
+/// Represents an instance of a running root isolate with window bindings. In
+/// normal operation, a single instance of this object is owned by the engine
+/// per shell. This object may only be created, used, and collected on the UI
+/// task runner. Window state queried by the root isolate is stored by this
+/// object. In cold-restart scenarios, the engine may collect this collect
```

Comment:
cold-restart is used in other places, but I think it's a bit incongruent with our more typical terminology at this point of hot reload/hot restart.

Also - `collect this collect` - just collect this?

Reply:
> cold-restart is used in other places, but I think it's a bit incongruent with our more typical terminology at this point of hot reload/hot restart.

I didn't know we settled on that terminology already. When did we do that? I'll change all references to cold-restart in a subsequent patch.

Reply:
> Also - collect this collect - just collect this?

Patching.